lib/elixir_auth_google.ex

```diff
@@ -29,10 +29,21 @@ defmodule ElixirAuthGoogle do
     "#{Atom.to_string(s)}://#{h}:#{p}"
   end
 
+  # When deployed to Fly.io the scheme is "http" and port is "80" ...
+  # This results in an incorrect url see:
+  # github.com/dwyl/elixir-auth-google/issues/94
+  def get_baseurl_from_conn(%{host: h, req_headers: r}) do
+    "#{Map.get(r, "x-forwarded-proto")}://#{h}"
+  end
+
   def get_baseurl_from_conn(%{host: h, scheme: s}) do
     "#{Atom.to_string(s)}://#{h}"
   end
 
+  def get_baseurl_from_conn(url) when is_binary(url) do
+    "https://#{url}"
```

Comment:
`Endpoint.url()` already includes the scheme, maybe it would be better to return the url as is?

Reply:
OK. Thanks for informing us. Will remove now. 👌

Reply:
Resolved by:

https://github.com/dwyl/elixir-auth-google/blob/8cc5e8818c77bd05e3a52ddd4586d5f4c4d31ea1/lib/elixir_auth_google.ex#L60-L73

Reply:
Updated. Thanks.